ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -924,10 +924,7 @@ GetTFToNgOpMap() {
       {"Square", {std::make_shared<opset::Multiply>()}},
       {"SquaredDifference", {std::make_shared<opset::SquaredDifference>()}},
       {"Squeeze", {std::make_shared<opset::Squeeze>(), constant}},
-      {"StridedSlice",
-       {std::make_shared<ngraph::op::Reverse>(),
-        std::make_shared<ngraph::op::Slice>(),
-        std::make_shared<ngraph::op::Reshape>()}},
+      {"StridedSlice", {std::make_shared<opset::StridedSlice>()}},
```

Comment:
constant is missing

Reply:
Because the translation uses it and I thought this map should contain all the ops used for translation or else it complains which is why we mention Constant at other places too.

Reply:
This map is only used to check if the op is supported by the backend, right? Besides, Constant is typically supported by all backends so checking it for each op translation is probably redundant.

I added it for the sake of completeness though.